src/DatePicker/DatePickerNavbar.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import Box from '../Box';
+import Button from '../Button';
+import Flex from '../Flex';
+import Icon from '../Icon';
+
+const DatePickerNavbar = ({ onPreviousClick, onNextClick, className }) => (
+  <Flex className={className} alignItems="center">
+    <Button
+      iconStart={<Icon color="icon.default" name="chevron" rotation="90" />}
+      onClick={() => onPreviousClick()}
```

Comment:
Would this be the same as `onClick={onPreviousClick}`?

Reply:
The scope changes by passing it directly.